cluster/service/cluster_service.go

```diff
@@ -62,15 +63,14 @@ func (c clusterService) CreateOrSaveClusterFromConfig(ctx context.Context) error
 			AuthClientSecret:  configCluster.AuthClientSecret,
 			AuthDefaultScope:  configCluster.AuthClientDefaultScope,
 		}
-
 		err := c.ExecuteInTransaction(func() error {
 			return c.Repositories().Clusters().CreateOrSave(ctx, rc)
 		})
-
 		if err != nil {
 			return err
 		}
 	}
+	log.Warn(ctx, map[string]interface{}{}, "done creating/updating clusters from config file")
```

Comment:
How about `creating/updating clusters from config file has been completed/done`?

Reply:
I can change the message if you prefer that wording

Reply:
done in e99d1ef